test/Chirp.ChirpInfrastructure.Tests/UnitTests.cs

```diff
@@ -0,0 +1,22 @@
+using Xunit;
+using ChirpInfrastructure;
+
+namespace Chirp.ChirpInfraStructure.Tests{
+
+public class UnitTest
+{
+    
+    [Fact]
+    public void TestThatTestWork()
+    {
+       //Arrange
+        string h = "Hej";

```

Comment:
More of a for-funsies comment than a problem comment, but we could rename the string to be "Hello World!" as that's a classic programming sentence. But again it doesn't change the function of the test, so more of a nice to have than need to have.